src/plugins/console/server/lib/spec_definitions/json/overrides/indices.simulate_index_template.json

```diff
@@ -0,0 +1,16 @@
+{
+  "indices.simulate_index_template": {
```

Comment:
Based on the [docs](), this should be:

```json
{
  "indices.simulate_index_template": {
    "data_autocomplete_rules": {
      "index_patterns": [],
      "version": 0,
      "template": {
        "mappings": { "__scope_link": "put_mapping" },
        "settings": { "__scope_link": "put_settings" },
        "aliases": {
          "__template": {
            "NAME": {}
          }
        }
      },
      "composed_of": [],
      "priority": 0,
      "_meta": {}
    },
    "patterns": [
      "_index_template/_simulate/{template}"
    ]
  }
}

```

Reply:
I reviewed the docs and see what you mean. Thanks.